packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/index.js

```diff
@@ -79,7 +101,9 @@ class TwoStepVerificationContainer extends React.PureComponent {
     this.props.modalActions.showModal('ConfirmDisable2FA', {
       authName: this.state.authName
     })
-    this.props.updateUI({ editing: false })
+    this.setState({
```

Comment:
not your code but can you remove `handleDisableTwoStep` on line 95 since it is not used anywhere